src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/ParameterDescriptions.java

```diff
@@ -31,8 +31,8 @@
   public static final String FORMAT =
       "Output format geojson (for /groupBy/boundary resources only), csv, or json; default: json";
   public static final String PROPERTIES =
-      "List of possible property-groups added to each OSM-element: 'tags' and/or 'metadata' " +
-      "and/or 'contributionTypes' (only for the /contributions/{geometryType} endpoints); "
+      "List of possible property-groups added to each OSM-element: 'tags' and/or 'metadata' "
+      + "and/or 'contributionTypes' (only for the /contributions/{geometryType} endpoints); "
```

Comment:
My IDE moves this line and the one below one step to the right when applying auto-formatting 🤔 
Checkstyle seems to be fine with both, the non-indented version and having an indentation 🤷‍♂️

Reply:
ok, let's make it consistent with the lines above. :+1: